swarm/storage/ldbstore.go

```diff
@@ -45,7 +44,7 @@ import (
 
 const (
 	gcArrayFreeRatio = 0.1
-	maxGCitems       = 5000 // max number of items to be gc'd per call to collectGarbage()
+	maxGCItems       = 5000 // max number of items to be gc'd per call to collectGarbage()
```

Comment:
Maybe increase to 500000, so that we remove 200MB max, and not 2MB max (when chunk size is 4KB)